avm/res/synapse/workspace/main.bicep

```
@@ -74,7 +74,7 @@ param sqlAdministratorLogin string
 param sqlAdministratorLoginPassword string = ''
 
 @description('Optional. Git integration settings.')
-param workspaceRepositoryConfiguration object = {}
+param workspaceRepositoryConfiguration object ?
```

```suggestion
param workspaceRepositoryConfiguration object?
```

The formatter will not like this space 😄